backend/core/src/property/entities/property.entity.ts

```diff
@@ -139,6 +145,12 @@ export class Property {
   @IsNumber({}, { groups: [ValidationsGroupsEnum.default] })
   yearBuilt?: number | null
 
+  @Column({ type: "text", nullable: true })
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  phoneNumber?: string | null
```

Comment:
Is this different than the leasingAgentPhone in this file? There is also a phoneNumber at in property.entity. I see in the [attached bug](https://github.com/CityOfDetroit/bloom/issues/227#issuecomment-887839372) that we're differentiating between "Property Phone" and "Manager Phone". Could we add some comments to clarify the difference here?

Reply:
Great question - I'm not clear on the difference between "Property Phone" and "Manager Phone" myself (or whether they can be or are often the same?). I've added a note to our Property Management agenda for next week to discuss this with Safiya. I'll wait on adding comments until I actually understand what these fields mean.